server/src/test/java/io/crate/metadata/upgrade/MetadataIndexUpgraderTest.java

```diff
@@ -86,6 +86,32 @@ public void test__all_is_removed_from_mapping() throws Throwable {
         assertThat(mapping.source().string(), Matchers.is("{\"default\":{\"properties\":{\"name\":{\"type\":\"keyword\",\"position\":1}}}}"));
     }
 
+    @Test
+    public void test__dropped_0_is_removed_from_mapping() throws Throwable {
+        IndexMetadata indexMetadata = IndexMetadata.builder(new RelationName("doc", "users").indexNameOrAlias())
+            .settings(Settings.builder().put("index.version.created", VersionUtils.randomVersion(random())))
```

Comment:
Seeing this, I had a thought of checking the `version.created` of the table and only proceed with checking for `_dropped_0` if it's `5.5.0` or `5.5.1`, but the check is simple so I believe it doesn't worth more complexity.